vm.go

```diff
@@ -538,8 +543,13 @@ func (session *ioSession) SendTerminalSize(columns, rows int) error {
 
 // SendSignal
 func (session *ioSession) SendSignal(signal syscall.Signal) error {
+	containerID := session.containerID
```

Comment:
Session is a 1:1 object with token. So the session object for the container workload is not the same as the session for an exec process (they both have their own token). So, the newcontainer relocation will only happen on the container session.